### PR TITLE
Add MO command to PyTorch conversion documentation

### PR DESCRIPTION
**Pull request by helena-intel**
Currently it seems more complicated than it is to convert a PyTorch model because you have to open multiple documents. Model Optimizer from ONNX is very simple, so I added a small section on how to do this, and added a line that you can load ONNX directly too.

I also removed the asterisks from this page.

### CODE REVIEW

**Review by kblaszczak-intel — APPROVED** [review]
I see no issues with the changes.

**Comment by helena-intel** [comment]
> I see no pressing issues with the changes. I can go through the text and try to improve it a bit (e.g. instead of "models that are tested" we could use "models that have been tested" or "a list of tested models" so that it cannot be understood in any other way) if you think it advisable.

All improvements would be much appreciated!

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md [hunk 1]
```diff
@@ -69,8 +69,23 @@ dummy_input = torch.randn(1, 3, 224, 224)
...
+```
+
+Replace `mo` with `python /path/to/model_optimizer.py` or `python3 /path/to/model_optimizer.py` for other OpenVINO installations.
+
+See [Convert an ONNX model](Convert_Model_From_ONNX.md) for more information and advanced options.
+
 ## Known Issues
 
-* Not all PyTorch\* operations can be exported to ONNX\* opset 9 which is used by default, as of version 1.8.1.
+* Not all PyTorch operations can be exported to ONNX opset 9 which is used by default, as of version 1.8.1.
```
Comment (kblaszczak-intel) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md:89]:
```suggestion
* Not all PyTorch operations can be exported to ONNX opset 9, which, as of version 1.8.1., is used by default.
```

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md [hunk 2]
```diff
@@ -69,8 +69,23 @@ dummy_input = torch.randn(1, 3, 224, 224)
...
 It is recommended to export models to opset 11 or higher when export to default opset 9 is not working. In that case, use `opset_version`
```
Comment (kblaszczak-intel) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md:90]:
```suggestion
It is recommended to export models to opset 11 or higher when export to the default opset 9 does not work. In that case, use the `opset_version`
```

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md [hunk 3]
```diff
@@ -69,8 +69,23 @@ dummy_input = torch.randn(1, 3, 224, 224)
...
 option of the `torch.onnx.export`. For more information about ONNX* opset, refer to the [Operator Schemas](https://github.com/onnx/onnx/blob/master/docs/Operators.md).
```
Comment (kblaszczak-intel) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md:91]:
```suggestion
option of `torch.onnx.export`. For more information about ONNX opset, refer to the [Operator Schemas](https://github.com/onnx/onnx/blob/master/docs/Operators.md).
```

**Review by kblaszczak-intel — COMMENTED** [review]
As requested, a more in-depth review with a bunch of suggestions and a few corrections. In case of any questions or doubts... Or constructive criticism ;) I'm all yours.

**Comment by azhogov** [comment]
please create PR from fork

**Pull request closed without merging**